src/com/palmergames/bukkit/towny/command/NationCommand.java

```diff
@@ -110,10 +111,25 @@
 		"tag"
 	));
 	
-	public static final List<String> nationToggleTabCompletes = new ArrayList<>(Arrays.asList(
-		"neutral",
+	static final List<String> nationToggleTabCompletes = new ArrayList<>(Arrays.asList(
+		"peaceful",

```

Comment:
The original command was neutral, and someone thought this wasn't quite clear enough and changed it to peaceful. Both should be available.

Reply:
If both are available, then should both be show in the `/n toggle` help? This may be confusing to users because there are two different words that do the same thing. What if the `/n toggle` looked like this?
![image](https://user-images.githubusercontent.com/55036602/74989789-6a20d280-540f-11ea-9b86-4a0b3d767733.png)

Reply:
make it so